src/index.html

```diff
@@ -1,16 +1,56 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search" 
+      data-qa="big" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon" 
+          class="icon icon--big" 
```

Comment:
```suggestion
          class="icon" 
```
you can combine all styles in the common class `icon`, and override necessary styles for other cases. 
Please check Checklist P2